BackEnd/Data/MySQLMethods.cs

```diff
@@ -100,50 +99,152 @@ public void Update(User userParam)
             _context.SaveChanges();
         }
 
+        public IEnumerable<object> TasksPerUsers(int id)
+        {
+            var tasks = _context.TasksPerUsers.Where(x => x.UserId == id);
+            return tasks;
+        }
+
+        // Projects Handling //
+        public IEnumerable<ProjectModel> GetAllProjects()
+        {
+            var result = _context.Projects.Select(p =>
+            new ProjectModel
+            {
+                Id = p.Id,
+                Title = p.Title,
+                Description = p.Description,
+                Tasks = p.Tasks.Select(s => new TaskModel
+                {
+                    TaskId = s.TaskId,
+                    Description = s.Description,
+                    Status = s.Status,
+                    Importance = s.Importance,
+                    ProjectId = s.ProjectId,
+                    UserId = s.UserId ?? default(int)
```

Comment:
Default(int) will result into 0.  Is this what you want? Are you sure there is no user with Id 0 ?